manifests/pulp.pp

```diff
@@ -146,7 +146,7 @@
     enable_rpm             => $katello::params::enable_yum,
     enable_iso             => $katello::params::enable_file,
     enable_deb             => $katello::params::enable_deb,
-    enable_puppet          => $katello::params::enable_puppet,
+    enable_puppet          => $katello::params::_enable_puppet,
```

Comment:
This part doesn't make a lot of sense. By definition we have Pulp 2 support at this point.
```suggestion
    enable_puppet          => $katello::params::enable_puppet,
```